### PR TITLE
New article: The Pantheon Deploy Log

### PR DESCRIPTION
**Pull request by rachelwhitton**
This PR closes issue #413 by creating a new doc: The Pantheon Deploy Log

@suzannealdrich and @nataliejeremy can you review? Thanks!

### CODE REVIEW

File: source/docs/articles/sites/deploy-log.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: The Pantheon Deploy Log
```
Comment (bmackinney) [inline source/docs/articles/sites/deploy-log.md]: "Deploying Code from Dev to Test to Live" (goal oriented titles > tool oriented titles)

File: source/docs/articles/sites/deploy-log.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+**Examples**:
+
+- Fix for issue-12345
+- New feature: User Bio Fields
+- Update contrib modules to 2.x experimental branches
+- Verified passes user acceptance tests  
+
+## Deploy to Test
+Each site on Pantheon comes with three environments: Dev, Test, and Live. The Deploy Log found on the Test environment allows you to push code up from Dev, and pull content down from Live.
```
Comment (bmackinney) [inline source/docs/articles/sites/deploy-log.md]: `while optionally pulling the database and files down from Live` - Important to make the distinction between DB/Files and "content" as the DB has more than just content in it.

File: source/docs/articles/sites/deploys.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+On WordPress site dashboards, cloning the content will expose an option to convert URLs from the Live environment's pattern to the Test environment's, including the protocol from HTTPS to HTTP for encrypted live environments.
+
+Drupal site deploys come with the ability to run `update.php` after  code deploys. `hook update n` and any other necessary database updates will be made automatically on the Test environment. 
+![Deploy Log Test Environment](/source/docs/assets/images/deploy-log-test-env.png)
+
+## Deploy to Live
+The Deploys tool in the Live environment deploys only the code from the Test environment. If you cloned the database and files from the Live environment when you deployed to Test, your Live environment will behave exactly as the Test environment did prior to the deploy. Deploys are performed by adding a git tag to the last commit on the Test environment. 
+All sites have the option of clearing all caches after deploying code. 
+Drupal sites have the option of running update.php after deploying code.
+![Deploy Log Live Environment](/source/docs/assets/images/deploy-log-live-env.png)
```
Comment (bmackinney) [inline source/docs/articles/sites/deploys.md]: Images are incongruent. This is a Drupal site, while the other is a WordPress site.

**Comment by rachelwhitton** [comment]
@bmackinney I removed the first screenshot - I think enough is communicated in suzanne's image. Is this okay to merge and ship?

**Pull request merged**